main.nf

```diff
@@ -109,34 +91,50 @@ process fusioncatcher {
 
     input:
     set val (name), file(reads) from fusioncatcher_reads
+    file fusioncatcher_data_dir from fusioncatcher_data_dir.collect()
 
     output:
     file '*.{txt,log,zip}' into fusioncatcher
-
     when: params.fusioncatcher
 
     script:
+    if (params.singleEnd) {
+    
     """
+    mkdir ${reads}_data
+    mv ${reads} ${reads}_data/
```

Comment:
Is the directory name important? May be safer to just call it `data`..

Reply:
Yes, I think so. If I recall correctly a lot of the output it always called the same and then it's nice to have the sample name in the directory to know what you are looking at. Especially while debugging. I think..

Reply:
Fair enough 👍